.travis.yml

```diff
@@ -53,6 +52,7 @@ jobs:
     - python: 3.4.3
```

Comment:
per 58b9f416534e , 3.4.3 is now supported, so this could be part of the problem -- it should be 3.4.1?

Reply:
Ah, yes. :man_facepalming:

Reply:
But I have no idea, why pip-9 wasn't getting installed on 3.4.3